src/root/views/deployments.js

```diff
@@ -267,6 +265,12 @@ class Deployments extends SFPComponent {
 
   render () {
     const { strings } = this.context;
+    const pending = this.isPending(this.props.eruOwners);
+    if (pending) {
```

Comment:
I think we can directly do
```
<App className='page--deployments'>
    { pending && <NewGlobalLoading /> }
    ...
```

Reply:
So the problem with that was that the rest of the render depends on variables that are not available yet and that causes errors. I could go through on each page and make sure that things don't error, but that seems a bit brittle, and in some cases it seems much safer to short-circuit the return and not attempt to render the main view. Any suggestions here for doing this better? This didn't seem great to me, but could not figure a better way without getting deep into each view to ensure it doesn't error if data isn't available yet.

Reply:
So for that what we can do is conditionally call the `renderContent()`. Something like
```jsx
<App className='page--deployments'>
    ...
    { this.props.eruOwners?.fetched && this.renderContent() }
```

Reply:
The reason we want to render `App` is that it'll show the headers. If we don't it'll look similar to how emergencies page currently load (i.e. without header / navbar)